src/utils/runtime/src/lib.rs

```diff
@@ -96,8 +95,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
         // Configure log output to stdout
         let fmt_layer = tracing_subscriber::fmt::layer()
             .compact()
-            .with_ansi(settings.colorful)
-            .with_timer(time::OffsetTime::local_rfc_3339().expect("could not get local offset!"));
+            .with_ansi(settings.colorful);
```

Comment:
May I ask whether this reversion of #6038 is intended? If not, I'll add it back.